addons/mail_group/tests/test_mail_group.py

```diff
@@ -78,6 +78,40 @@ def test_find_member(self):
         member = self.test_group._find_member(email, partner_2.id)
         self.assertFalse(member, 'Should not return any member because the only one with the same email has a different partner')
 
+    def test_find_member_for_alias(self):
+        """Test the matching of a mail_group.members, when 2 users have the same partner email, and
+        that the first user was subscribed."""
+        User = self.env['res.users'].with_context(no_reset_password=True)
+
+        alias = self.env['mail.alias'].create({
+            'alias_name': 'test_find_member_for_alias',
+            'alias_model_id': self.env.ref('base.model_res_partner').id,
+            'alias_contact': 'followers',
+        })
+        user = self.user_portal
+        User.create({
+            'login': 'login_2',
+            'name': 'login_2',
+            'email': user.email,
+        })
+
+        member = self.env['mail.group.member'].create({
+            'partner_id': user.partner_id.id,
+            'mail_group_id': self.test_group.id,
+        })
+        self.assertEqual(member.email, user.email)
+
+        # Should not return string (exception) if at least one members have the same email, whatever
+        # the user_id (author_id) that could match this email.
+        #
+        # if return None, False it means no error
+        # if return string (e.g. Only members can send email to the mailing list.), it means an error occurs
+        msg_dict = {
+            'author_id': self.env['mail.thread']._mail_find_user_for_gateway(user.email).id,
```

Comment:
I really think author_id is a partner, not a user
![screenshot_spect](https://user-images.githubusercontent.com/54316226/143247800-0a2179b6-1376-45b6-9195-ad078c342b3d.png)

Reply:
ok my bad, _mail_find_user_for_gateway return a user, but alias_get_error_message take a partner_id.
It had no influence since this test tests that we ignore this value... but yes you are right, I updated.